packages/llama-cpp/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ggerganov/llama.cpp
+TERMUX_PKG_DESCRIPTION="Port of Facebook's LLaMA model in C/C++"
+TERMUX_PKG_LICENSE=GPL-3.0
+TERMUX_PKG_MAINTAINER=@termux
+TERMUX_PKG_VERSION=294f424
```

Comment:
This version is not valid. I suggest using the commit date or commit position as the package version.